procedures/accounts_setup.rst

```diff
@@ -0,0 +1,19 @@
+Here is a list of accounts and access needed to be successful in Release Engineering
+
+General:
+ * `JIRA <https://jira.mozilla.com/projects/RELENG/issues>`_ - add username to Mozilla instance
```

Comment:
Nit: s/Mozilla/Releng and s/instance/project/. Based on my interactions so far, SSO is synced-up in JIRA, we just need to ensure we add roles for those accounts in our dedicated RelEng project.